README.md

```diff
@@ -10,6 +10,11 @@ npm run dev
 yarn dev
```

Comment:
Since the package manager in-use in this repo is `pnpm` we can update the docs to have `pnpm dev`